static/files/benchmark1.json

```diff
@@ -1,44 +1,40 @@
 {
     "informations": {
-        "system": { // system informations
```

Comment:
The second file is not about being cleaned, but it has a list of two entries for each spot. I wanted to see if I got everything correct when displaying more than one result.